release_draft.md

```diff
@@ -0,0 +1,303 @@
+[ignition-release](https://github.com/ignition-release/) GitHub organization.
+(see [release repositories document](releasing/release_repositories.md) for more
+information about how they are used).
+
+It is required to clone the corresponding release repository to update the
+binary version:
+
+```bash
+# Ignition library named foo and major version X
+git clone https://github.com/ignition-release/ign-fooX-release
+```
+
+To bump the package versions that will appear in Debian/Ubuntu binary packages
+there is a helper script in `release-tools` (see
+[prerequisites](#prerequisites)). The script is called `changelog_spawn.sh` and
+require to be executed while the active directory is a `release repository`:
+
+```bash
+# Ignition library named foo and major version X
+cd ign-fooX-release
+~/release-tools/release-repo-scripts/changelog_spawn.sh X.Y.Z-R
+
+# Example ign-cmake2 bumped from 2.0.0 to 2.0.1
+cd ign-cmake2-release
+~/release-tools/release-repo-scripts/changelog_spawn.sh 2.0.1-1
+```
+
+`changelog_spawn.sh` will display information about the Ubuntu/Debian versions
+being updated as well as a `git diff` before uploading information to the GitHub
+release repository.
+
+## Launch the release in the building server
+
+After updating the code and releasing metadata everything is ready to launch the
+build in the server. Different actions needs to happen:
+
+ 1. Generate a tarball with the Ignition library sources corresponding to the new
+    version in the Ignition developer local system. Upload the tarball to S3 cloud storage.
+ 1. Request `build.osrfoundation.org` server to start the jobs for:
+   1. Debian/Ubuntu: use `ign-fooX-debbuilder` job names
+   1. Brew: entry job is `generic-release-homebrew_pull_request_updater`
+
+The `release.py` script perform with all these actions.
+
+### Executing release.py
+
+`release.py` works from Ignition library code where the new version has been
+set. Be sure to run it from a clone of a permanent branch (`ign-fooX` typically)
+since some git tags will be generated and uploaded.
+
+From the source code directory execute the `release.py`. You will need the token
+described in the [credentials section](#credentials).
+
+```bash
+# Ignition library named foo and major version X
+cd ign-fooX
+git checkout ign-fooX
+
+# Example ign-cmake2 bumped to 2.0.1 with test_token credential
+cd ign-cmake2
+git checkout ign-cmake2
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 test_token
+```
+
+#### dry-run simulation mode
+
+`release.py` tool supports a `--dry-run` flag that allows to simulate releases
+(nothing is modified) to help with safe checks done by the tool and providing
+the user with the final calls that will be done to the server.
+
+The script needs to be run from the library repository:
+
+```bash
+# Example of dry-run for ign-cmake2 bumped to 2.0.1
+cd ign-cmake2
+git checkout ign-cmake2
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 test_token --dry-run
```

Comment:
```suggestion
# Example of dry-run for ign-cmake2 bumped to 2.0.1
cd ign-cmake2
git checkout bump_to_2_0_1
~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 test_token --dry-run
```

Reply:
We always release from supported branches (i.e: ign-cmake2, ign-transport8). The `bump_to_X_Y_Z` branch is used in the "Update code Version" section which detailed that a new PR should be open and merged (see "Update binary version").